CONTRIBUTING.rst

```diff
@@ -0,0 +1,74 @@
+Contribution Guidelines
+=======================
+
+We use `Github projects`_ to organize our ticket workflow.
+If you want to lend a hand, check out the current project and choose one of the tickets from the issues!
+If there's `a particular issue`_ you would like to work on and it isn't already assigned, leave a comment on that issue indicating your desire to work on it.
+Then, start working!
+
+Start Developing
+----------------
+
+To get started with your contribution, fork this project into your own GitHub profile and clone that forked repository to your machine.
+As soon as you finish cloning, navigate into the new directory and set the original project repository as the remote name `upstream`.
```

Comment:
"upstream" needs double backticks in rst.

I think this can be reworded as:
**After cloning**, navigate into the new directory and set the original project repository as the remote name**d** ``upstream``.